### PR TITLE
Update to molo V4.4.0

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@codiebeulaine @nathanbegbie @Ntu2koTrevor tiny review :)

File: tuneme/settings/base.py [hunk 1]
```diff
@@ -180,10 +180,10 @@
...
 CELERYBEAT_SCHEDULE = {
     # Executes every morning at 8:00 A.M GMT+2
-    'add-every-morning': {
-        'task': 'molo.profiles.task.send_user_data_to_slack',
-        'schedule': crontab(minute=0, hour=8)
-    },
+    # 'add-every-morning': {
+    #     'task': 'molo.profiles.task.send_user_data_to_slack',
+    #     'schedule': crontab(minute=0, hour=8)
+    # },
```
Comment (nathanbegbie) [inline tuneme/settings/base.py:186]: Do you want to comment out or delete? I think we're deprecating this feature anyway?

**Comment by codiebeulaine** [comment]
👍

File: tuneme/settings/base.py (hunk 1 above)
Comment (smarzban) [reply tuneme/settings/base.py:186]: There are still discussions around this to either improve it or deprecate it, so I thought maybe lets comment it out for now and see what would be the final decision.

**Pull request merged**